### PR TITLE
Replace CryptoJS by WebCrypto

### PR DESCRIPTION
**Pull request by sadortun**
You'll have to :
- delete transactions/accounts
- clear  user ClientKey
- logout/login

When starting the client, you will have a Chrome warning about insecure site ( because of the Vite self-signed certificate ).  
you will also need to `Allow Insecure connections` for your localhost

image

image

### CODE REVIEW

File: common/Crypto/cryptoJs.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
+// /**
```
Comment (mkitzmann) [inline common/Crypto/cryptoJs.ts:1]: do we want to keep this?

File: frontend/src/parseConfig.ts [hunk 2]
```diff
@@ -20,3 +20,54 @@ window.Parse.enableLocalDatastore()
 Parse.CoreManager.setStorageController(Parse.IndexedDB)
 
 
+// import { SecureObject } from '/@common/models/base/SecureObject'
```
Comment (mkitzmann) [inline frontend/src/parseConfig.ts:23]: also this?

File: frontend/src/store/HoldingStore.ts [hunk 3]
```diff
@@ -13,20 +13,18 @@ export const useHoldingStore = defineStore({
   id: 'holding',
 
   state: () => ({
-    liveSubscription : null,
-    holdings         : [],
+    subscriptionPromise : null,
```
Comment (mkitzmann) [inline frontend/src/store/HoldingStore.ts:16]: maybe we should add a type for this store as well

**Review by mkitzmann — CHANGES_REQUESTED** [review]
We should probably adjust the config.default

File: frontend/src/parseConfig.ts (hunk 2 above)
Comment (sadortun) [reply frontend/src/parseConfig.ts:23]: This is the way i originally wanted to implement it, but it's not working. I'll have to see with `Parse-JS` to see whats going on.

Even if we set our `ObjectController`, it's not called by the SDK

File: common/Crypto/cryptoJs.ts (hunk 1 above)
Comment (mkitzmann) [reply common/Crypto/cryptoJs.ts:1]: fuck ie11 ;)

File: frontend/src/parseConfig.ts (hunk 2 above)
Comment (mkitzmann) [reply frontend/src/parseConfig.ts:23]: ah, ok

**Comment by sadortun** [comment]
> We should probably adjust the config.default

Good catch, done.

**Review by mkitzmann — APPROVED** [review]

**Pull request merged**